packages/validator/templates/configSecurity.json

```diff
@@ -0,0 +1,98 @@
+{
+  "config": {
+    "ois": {
+      "__arrayItem": {
+        "__oisSpecs": {}
+      }
+    },
+    "triggers": {
+      "__optional": {
+        "request": {
+          "__arrayItem": {
+            "endpointId": {
+              "__regexp": "^0x[a-fA-F0-9]{64}$"
+            },
+            "oisTitle": {},
+            "endpointName": {}
+          }
+        }
+      }
+    },
+    "nodeSettings": {
+      "providerIdShort": {
+        "__regexp": "^[a-fA-F0-9]{7}$"
+      },
+      "nodeVersion": {
+        "__regexp": "^([0-9]*\\.){2}([0-9]*)$"
+      },
+      "cloudProvider": {
+        "__regexp": "^aws$"
+      },
+      "region": {},
+      "stage": {},
+      "logFormat": {
+        "__regexp": "^(json|plain)$"
+      },
+      "chains": {
+        "__arrayItem": {
+          "id": {},
+          "type": {
+            "__regexp": "^evm$"
+          },
+          "providers": {
+            "__arrayItem": {
+              "name": {},
+              "url": {
+                "__regexp": "^(https?|ftp)://[^\\s/$.?#].[^\\s]*$"
+              },
+              "__optional": {
+                "blockHistoryLimit": {},
+                "minConfirmations": {}
+              }
+            }
+          },
+          "__optional": {
+            "providerAdminForRecordCreation": {
+              "__regexp": "^0x[a-fA-F0-9]{40}$"
+            },
+            "contracts": {
+              "__keyRegexp": "^(Airnode|Convenience)$",
```

Comment:
According to the specs, it must include *both* `Airnode` and `Convenience`, so this should be checked for separately, see: https://github.com/api3dao/api3-docs/blob/8e41ab2597cb19cb75e4904d6e9b214287c1d738/airnode/config-json.md#nodesettingschains

Reply:
Updated in 1d40c07